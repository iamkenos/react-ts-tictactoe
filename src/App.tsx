import React from "react";
import Square from "./components/Square/Square";

class App extends React.Component {
  render() {
    return (
      <Square/>
    );
  }
}

export default App;
