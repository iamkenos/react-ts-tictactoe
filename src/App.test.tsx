import { render } from "@testing-library/react";
import App from "./App";

describe("Application", () => {
 let fragment: DocumentFragment;
  it("should render the app", () => {
    givenRenderApp();
    thenAppSnapshotMatch();
  });

  function givenRenderApp() {
    const { asFragment } = render(<App />);
    fragment = asFragment();
  }

  function thenAppSnapshotMatch() {
    expect(fragment).toMatchSnapshot();
  }
});