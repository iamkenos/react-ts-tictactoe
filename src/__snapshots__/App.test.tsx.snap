// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application should render the app 1`] = `
<DocumentFragment>
  <button
    class="square"
  >
    X
  </button>
</DocumentFragment>
`;
